app/routes/missions/icecube.mdx

```diff
@@ -5,63 +5,158 @@ handle:
 
 import logo from '~/img/icecube-logo.png'
 
-# IceCube
+# IceCube Neutrino Observatory
 
 <img
   src={logo}
   alt="IceCube logo"
-  width="640"
-  height="181"
+  align="right"
+  width="150"
+  height="135"
   className="height-auto"
 />
 
-**Construction Completion Date:** December 17, 2010
-
-**Planned Lifetime:** IceCube Upgrade planned for Winter 2025
+**Construction Completion Date:** December 2010
 
 **End of Operations:** No specific requirement
 
-**Data Archives:**
-https://icecube.wisc.edu/science/data-releases/
+**Data Archives:**\
+https://dataverse.harvard.edu/dataverse/icecube \
+https://icecube.wisc.edu/science/data-releases/ \
 https://heasarc.gsfc.nasa.gov/W3Browse/icecube/icecubepsc.html
```

Comment:
Could we turn this into a bulleted list with actual descriptive link text, rather than just bare URLs?